src/index.html

```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="block-stars__stars"></div>
```

Comment:
children should be as an elements of stars block (according to BEM methodology)
And use singular name for you star
Apply all over the project
```suggestion
      <div class="stars__star"></div>
```